src/main/scala/com/codacy/parsers/implementation/CloverParser.scala

```diff
@@ -0,0 +1,71 @@
+package com.codacy.parsers.implementation
+
+import java.io.File
+
+import com.codacy.api.{CoverageFileReport, CoverageReport}
+import com.codacy.parsers.CoverageParser
+import com.codacy.parsers.util.{TextUtils, XMLoader}
+
+import scala.util.{Failure, Success, Try}
+import scala.xml.{Elem, Node, NodeSeq}
+
+object CloverParser extends CoverageParser {
+  private val COVERAGE = "coverage"
+  private val PROJECT = "project"
+  private val METRICS = "metrics"
```

Comment:
Follow [Scala naming conventions](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) for constants

Reply:
The names aren't so explicative either... `CoverageTagName`? I don't know what's better

Reply:
Done. I replaced them with <name>Tag. You're right it's clearer. Let me know if you think we could improve on this